src/Microsoft.AspNet.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -29,10 +29,10 @@ public GeneratedTagHelperContext()
             MarkAsHtmlEncodedMethodName = "Html.Raw";
             StartTagHelperWritingScopeMethodName = "StartTagHelperWritingScope";
             EndTagHelperWritingScopeMethodName = "EndTagHelperWritingScope";
-            RunnerTypeName = "TagHelperRunner";
-            ScopeManagerTypeName = "TagHelperScopeManager";
-            ExecutionContextTypeName = "TagHelperExecutionContext";
-            TagHelperContentTypeName = "TagHelperContent";
+            RunnerTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperRunner";
+            ScopeManagerTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperScopeManager";
+            ExecutionContextTypeName = "Microsoft.AspNet.Razor.Runtime.TagHelperExecutionContext";
+            TagHelperContentTypeName = "Microsoft.AspNet.Razor.TagHelperContent";
```

Comment:
These are type names. adding the `global::` is more of a code generation mechanic in my mind. That's how CodeDOM used to do it.